ornl/summit/summit.config.yaml

```diff
@@ -29,10 +29,8 @@ config:
 
   # Locations where different types of modules should be installed.
   module_roots:
-    tcl:     /gpfs/alpine/csc322/world-shared/spack-install/module-roots/modules
-    lmod:   /gpfs/alpine/csc322/world-shared/spack-install/module-roots/lmod
-    dotkit: /gpfs/alpine/csc322/world-shared/spack-install/module-roots/dotkit
-
+    tcl:    /ccs/home/xmeng/hpctoolkit-install/module-roots/modules
```

Comment:
same comment as above

Reply:
run groups on summit and let me know what groups you are in.

Reply:
```
[xmeng@login4.summit ~]$ groups
xmeng gen010
```